apple/RNCWebViewImpl.m

```diff
@@ -240,6 +253,37 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecogni
   }
 }
 
+#if USE_EDITMENU_IOS_16
+// Override the method startLongPress
+- (void)startLongPress:(UILongPressGestureRecognizer *)gestureRecognizer {
+    if (gestureRecognizer.state == UIGestureRecognizerStateBegan) {
+      UIEditMenuConfiguration *configuration = [UIEditMenuConfiguration configurationWithIdentifier:@"webViewEdit" sourcePoint:[gestureRecognizer locationInView:self]];
+      [self.editMenuInteraction presentEditMenuWithConfiguration:configuration];
+    }
+}
+
+- (UIMenu *)editMenuInteraction:(UIEditMenuInteraction *)interaction menuForConfiguration:(UIEditMenuConfiguration *)configuration suggestedActions:(NSArray<UIMenuElement *> *)suggestedActions {
+  NSMutableArray *actions = [suggestedActions mutableCopy];
+  NSMutableArray *customMenuActions = [NSMutableArray arrayWithCapacity:self.menuItems.count];
+
+    for(NSDictionary *menuItem in self.menuItems) {
+      NSString *menuItemLabel = [RCTConvert NSString:menuItem[@"label"]];
+      NSString *menuItemKey = [RCTConvert NSString:menuItem[@"key"]];
+      UIAction *item = [UIAction actionWithTitle:menuItemLabel image:nil identifier:nil handler:^(UIAction *action) {
+          [self tappedMenuItem:menuItemKey];
+      }];
+      [customMenuActions addObject: item];
+    }
+    UIMenu *customMenu2 = [UIMenu menuWithChildren:customMenuActions];
+
+  // [actions addObject:customMenu];
```

Comment:
Should this be removed?

Reply:
i see, i will update this

Reply:
Updated, please take a look. Thank you so much <3